### ISSUE TITLE
How to use GPUs for tabular

### ISSUE THREAD

**Issue by mli** [awslabs/autogluon#1097]
I didn't find a tutorial how to use leverage gpus for tabular

One related notebook is https://www.kaggle.com/innixma/autogluon-rapids-top-1 Though I feel it's a little hard to follow

```
'XGB': {'ag_args_fit': {'num_gpus': 1}},
'CAT': {'ag_args_fit': {'num_gpus': 1}},
'GBM': [{}, {'extra_trees': True, 'ag_args': {'name_suffix': 'XT'}}, 'GBMLarge'],
'NN': {'ag_args_fit': {'num_gpus': 1}},
'FASTAI': {'ag_args_fit': {'num_gpus': 1}},
```

for example:
1. unclear which model I can use gpu
2. i even don't know what models i should include

is there an easy option to enable gpu, and autogluon will try its best to put proper models into gpu?

**Comment by Innixma** [comment]
Yes, there is an easy option to enable GPU:

`predictor = TabularPredictor(...).fit(..., ag_args_fit={'num_gpus': 1})`

This will force all models capable of running on GPU to run on GPU. This includes XGB, CAT, GBM, NN, and FASTAI at present.

The reason the notebook looks a bit more complicated is that GBM was actually worse with GPU enabled for that dataset, so I had to specify gpu usage on a per-model basis.

I'll plan to add updated documentation on this functionality for `0.3` release.

**Comment by Innixma** [comment]
#1101 adds an FAQ section on using GPUs. Will plan to add further documentation once GPU usage is more solidified (ideally we want to automatically use GPUs when available, however there are some accuracy inconsistencies at present between CPU and GPU training as some models train differently depending on if they use GPU or CPU.
